packages/coordinator/src/navigation/CoordinatorRouter.tsx

```diff
@@ -75,35 +122,49 @@ export default function CoordinatorRouter() {
       appState.coordinator?.activeHospitalsForCoordinator!;
     content = (
       <Switch>
-        <Route exact path={"/" + CoordinatorScreen.SCHEDULED_APPOINTMENTS}>
-          <ManageAppointmentsScreenContainer
-            activeHospitalsForCoordinator={activeHospitalsForCoordinator}
-          />
-        </Route>
-        <Route exact path={"/" + CoordinatorScreen.DONORS}>
-          <SearchDonorsScreenContainer />
-        </Route>
-        <Route exact path={"/" + CoordinatorScreen.BOOKED_DONATIONS}>
-          <ScheduledAppointmentsContainer
-            activeHospitalsForCoordinator={activeHospitalsForCoordinator}
-          />
-        </Route>
-        <Route exact path={["/" + CoordinatorScreen.ADD_APPOINTMENTS, "*"]}>
-          <AddAppointmentsScreenContainer
-            activeHospitalsForCoordinator={activeHospitalsForCoordinator}
-          />
-        </Route>
+        {canViewOpenAppointments && (
```

Comment:
Do you think this is needed?
Do we need to block users from loading these pages?

I mean, if they manually go to that url without having permissions to do anything in it, I am ok with it.
I just think this makes this code a bit too complex

Reply:
I think it's good practice to not allow unauthorized people to manually navigate to a page they shouldn't see.
It maybe complicates the code (a tiny bit) but makes us not need to think about it.